### PR TITLE
Team Formation on StudentOverview

### PR DESCRIPTION
**Pull request by vaniaggarwal**
updated progress bar, moved styling sheet, created group request and team member requests ui

### CODE REVIEW

File: src/app/(auth-user)/my-team/page.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
-import styles from "./profile.module.css";
+import styles from "./studentprofile.module.css";
```
Comment (v3ceban) [inline src/app/(auth-user)/my-team/page.jsx]: Please move to other styles

File: src/components/Checklist.jsx [hunk 2]
```diff
@@ -1,52 +1,55 @@
...
+    };
 
     return (
         <div className="border-solid border-4 bg-gray-40 border-spacing-10 max-w-xs p-6 m-6">
-            <h1 className="text-lg font-bold"> Progress </h1>
-            <div className="mt-2"> 
+            <h1 className="text-lg font-bold">Progress</h1>
+            <div className="mt-2">
                 {items.map((item, index) => (
                     <div key={index} className="flex items-center space-x-2 mb-2">
```
Comment (v3ceban) [inline src/components/Checklist.jsx:31]: Index key might not be unique (if there are more than one component rendered that uses integers 0, 1, 2, 3... on page)

File: src/components/StudentAccountForm.jsx [hunk 3]
```diff
@@ -7,8 +7,10 @@ import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
 import { Camera } from "lucide-react"; // importing camera icon from Lucide icons
 import styles from "@/styles/StudentAccountForm.module.css";
 import { usePathname } from "next/navigation";
+import {useSession } from "next-auth/react"; // added to import session data 
 
 const StudentAccountForm = ({ user, userUpdate, hideInstruction }) => {
+  const {data: session } = useSession(); // access session data
```
Comment (v3ceban) [inline src/components/StudentAccountForm.jsx:13]: no need to access session data here. session user is already passed to component in props

**Review by marleyyvon — APPROVED** [review]

File: src/components/FacultyAccountForm.jsx [hunk 4]
```diff
@@ -7,9 +7,11 @@ import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
 import { Camera } from "lucide-react"; // importing camera icon from Lucide icons
 import styles from "@/styles/FacultyAccountForm.module.css";
 import { usePathname } from "next/navigation";
+import { useSession } from "next-auth/react";
 
 
 const FacultyAccountForm = ({ user, userUpdate, hideInstruction }) => {
+  const {data: session } = useSession(); // added to access session data
```
Comment (v3ceban) [inline src/components/FacultyAccountForm.jsx]: No need to get session here. user prop already has it (view parent component that imports and renders FacultyAccountForm)

**Review by v3ceban — CHANGES_REQUESTED** [review]

**Pull request merged**